addons/im_livechat/static/tests/channel_invite_tests.js

```diff
@@ -32,3 +32,79 @@ QUnit.test("Can invite a partner to a livechat channel", async (assert) => {
     await click("button[title='Show Member List']");
     assert.containsOnce($, ".o-discuss-ChannelMember:contains(James)");
 });
+
+QUnit.test("Available operators come first", async (assert) => {
+    const pyEnv = await startServer();
+    pyEnv["res.partner"].create({
+        name: "Harry",
+        im_status: "offline",
+        user_ids: [pyEnv["res.users"].create({ name: "Harry" })],
+    });
+    const availableOperatorId = pyEnv["res.partner"].create({
+        name: "Ron",
+        im_status: "online",
+        user_ids: [pyEnv["res.users"].create({ name: "Available operator" })],
+    });
+    pyEnv["im_livechat.channel"].create({
+        available_operator_ids: [Command.create({ partner_id: availableOperatorId })],
+    });
```

Comment:
`availableOperatorId` => `ronId`
Test is much easier to read with short and brief naming :)